src/dune_pkg/rev_store.ml

```diff
@@ -449,25 +449,50 @@ let branch_of_refspec refspec =
     String.drop_prefix remote_ref ~prefix:"refs/heads/")
 ;;
 
+let find_section =
+  let re =
+    Re.seq
+      [ Re.(rep space)
+      ; Re.str "[remote"
+      ; Re.(rep space)
+      ; Re.char '"'
+      ; Re.group (Re.rep (Re.diff Re.any (Re.char '"')))
+      ; Re.char '"'
+      ; Re.(rep space)
```

Comment:
I checked and git doesn't support spaces between the closing double quote and the `]`.